docs/templates/applications.md

```diff
@@ -14,6 +14,7 @@ Weights are downloaded automatically when instantiating a model. They are stored
 - [VGG19](#vgg19)
 - [ResNet50](#resnet50)
 - [InceptionV3](#inceptionv3)
+- [InceptionV4](#inceptionv4)
```

Comment:
Can you also add this to the table under the `Documentation for individual models` header?